test/utils.py

```diff
@@ -1229,3 +1229,10 @@ def get_admin_connection_params(connection_type, project):
         ]
 
     assert False, "Unknown connection type: %s" % connection_type
+
+
+def find_file_in_path(path_variable, name):
```

Comment:
This name says me "I can find a file in a specfiied path", but in fact it uses `PATH` variable.

Reply:
BTW, look at `shutil.which`. Is it what you need?